src/logger.ts

```diff
@@ -0,0 +1,41 @@
+import * as winston from "winston";
+import { Logger } from "winston";
+import * as path from "path";
+import { LOGGER_LEVEL } from "./config";
+
+const errorFilePath = path.resolve("..", "logs", "error.logs");
+const combinedErrorFilePath = path.resolve("..", "logs", "combined.logs");
+
+const container = new winston.Container();
+
+const winstonOptions = {
+  level: LOGGER_LEVEL,
+  format: winston.format.json(),
+  defaultMeta: { "worker-id": "dfault-graphile-worker" },
```

Comment:
Typo; also would it be better to drop the `default-` anyway?

```suggestion
  defaultMeta: { "worker-id": "graphile-worker" },
```